### PR TITLE
ticket 4 first pr

### PR DESCRIPTION
**Pull request by EddySafc**

### CODE REVIEW

File: src/components/Reviews.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import { getReviews } from "../requests";
+
+const Reviews = ({ reviews, setReviews }) => {
+  useEffect(() => {
+    getReviews().then((data) => {
+      setReviews(data);
+    });
+  }, []);
+  return (
+    <section>
```
Comment (callumu21) [inline src/components/Reviews.jsx:11]: We should give users some visual feedback here for when the page is loading and waiting for the fetch request to complete.

**Review by callumu21 — CHANGES_REQUESTED** [review]
Good start! Just a few notes:

1. Using CSS, you should remove the bullet points in the list view of reviews. 
2. There are some images not coming through, so maybe add an alt tag to the image display in case that happens.
3. You should think about using an isLoading state to give users feedback about when your list of reviews is being fetched.

File: src/components/Reviews.jsx [hunk 2]
```diff
@@ -1,28 +1,39 @@
...
+          {reviews.map((review) => {
+            return (
+              <li key={review.review_id}>
+                <p>{review.title}</p>
+                <p>{review.owner}</p>
+                <p>{review.category}</p>
+                <img
+                  id="review-image"
+                  src={review.review_img_url}
+                  alt="no image available"
```
Comment (pkaodev) [inline src/components/Reviews.jsx:28]: no image available isn't super helpful.  Use some property that exists on the review.  What is the image meant to be portraying?

**Comment by pkaodev** [comment]
Once you've updated the alt text feel free to merge.

**Pull request merged**